React-Projects/job-portal/src/Login.jsx

```diff
@@ -1,7 +1,45 @@
-import React from "react";
+import React, { useState } from "react";
 import "./styles/Login.css";
 
 function Login() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  const loginForm = (event) => {
+    event.preventDefault();
+
+    var requestBody = {
+      email: email,
+      password: password,
+    };
+
+    fetch("http://127.0.0.1:8000/api/login", {
```

Comment:
Also, please take `http://127.0.0.1:8000/api/login` as cont somewhere because this will be needed at multiple places

Reply:
@abhi-codal Understood